rvsol/README.md

```diff
@@ -0,0 +1,22 @@
+# Asterisc VM contract
+
+## Deployment
+Currently, Asterisc only supports the local devnet launched from the Optimism monorepo.
+
+### Prerequisites
+1. Running local devnet launched from the Optimism monorepo
+   - Run ```make devnet-up``` in the monorepo root.
+   - Set the env var `DEPLOY_ETH_RPC_URL` to the L1 RPC endpoint
+   - Set the env var `CHAIN_DEPLOYMENT_FILE` to the path of devnet deployment file (`optimism/packages/contracts-bedrock/deployments/devnetL1/.deploy`)
+   - Set the env var `DEPLOY_CONFIG_PATH` to the path of devnet deploy config file (`optimism/packages/contracts-bedrock/deploy-config/devnetL1.json`)
+2. Asterisc absolute prestate of op-program
+   - Run ```make op-program-client-riscv``` in `optimism/op-program`
+   - Set the built elf file path to the env var `OP_PROGRAM_PATH`
+
+### How to deploy
+1. Build Asterisc binary and contracts
+   - Run ```make build``` in the project root
+2. Generate prestate proof
+   - Run ```make prestate``` in the project root
+3. Run deploy script
```

Comment:
We must `cd` into `rvsol` because of relative address.
```
DEPLOY_CONFIG_PATH=$LOCAL_DEPLOY_CONFIG_PATH forge script -vvv scripts/Deploy.s.sol:Deploy --rpc-url "$DEPLOY_ETH_RPC_URL" --broadcast --private-key "$DEPLOY_PRIVATE_KEY"
```
not
```
DEPLOY_CONFIG_PATH=$LOCAL_DEPLOY_CONFIG_PATH forge script -vvv rvsol/scripts/Deploy.s.sol:Deploy --rpc-url "$DEPLOY_ETH_RPC_URL" --broadcast --private-key "$DEPLOY_PRIVATE_KEY"
```

Reply:
Fixed the script not to use relative path, but anyway we have to run the script in the `rvsol` dir to use `forge` command